docs/meetings/team/2022_10_11.md

```diff
@@ -0,0 +1,76 @@
+# 🚀 Weekly Team Meeting Minutes
+
+**Start Time** 5:00pm
+
+**End Time** 6:08pm
+
+**Location** Discord
+
+**Date** Sept 16th 2022
+
+## 👋 Attendence
+
+1. Logan
+2. Veronica
+3. Matt
+4. Edouard
+
+***Attendence Comments***
+
+## 🧸 Check-In
+
+- Dad I got a hair cut ?
+  - Do you mean you got all of your hairs cut
+
+## Announcements
+
+- Requiments presentation slides due Oct 11
+- Requirements video due Oct 14th
+- Group evaluation is due Oct 14th
+- Requirements Report due Oct 21st
+- GitStats due Oct 14th
+- Veronica's Birthday Nov 1st
+
+## Last Week (Restrospective)
+
+- GitStats from last week may have fallen short
+  - Veronica would like to make sure stat are good earlier on friday.
+  - Does not mind turing in the stats
+- We did well on our presentation, Naziem was appricative
+
+## This Week  
+
+### Premium Access to Kukurella
+
+- One email address for each team
+- Need to contact Naziem for access
+
+### Sharing Repo
+
+- No descion made yet
+- Private repo ? Then add other members from Kukurella
+
+### Client Meeting
+
+- Going over requirements to solidify project
+- Establish what we need to accomplish and figure out what they already have
+- Access to the API
+
+### Video Presentation
+
+- May be recorded while presenting in-class
```

Comment:
Since we didn't record ourselves in class, we will need to make sure we get this done today. Will discuss after client meeting